Orchestrator.py

```diff
@@ -569,9 +570,11 @@ def orchestrate(self, action ):
 				self.sequenceTiers(Orchestrator.TIER_STOP)
 
 				for currTier in self.sequencedTiersList:
-				
+					logger.info("IgnoreStopValue for {} tier is {}".format(currTier, self.tierSpecDict[currTier][Orchestrator.TIER_STOP][Orchestrator.TIER_IGNORE_STOP]))
+					if 	self.tierSpecDict[currTier][Orchestrator.TIER_STOP][Orchestrator.TIER_IGNORE_STOP] =='True':
+						logger.info('Orchestrate() IgnoreStop set to True for Tier {} so no stop action was performed'.format(currTier))
```

Comment:
Keep this message as is.  Good to output as "info" here to ensure it shows in the logfile